### ISSUE TITLE
CLI does not accept parameters without a double dash

### ISSUE THREAD

**Issue by lavoiems** [Epistimio/orion#818]
**Describe the bug**
CLI requires the parameters to start with a double dash (--).

**Expected behavior**
CLI is more permissive to the way that the parameters are injected. For example, hydra requires some parameters without a double dash.

**Steps to reproduce**
Inject a parameter without a double dash.

**Environment (please complete the following information):**
 - Oríon version: 0.2.2

**Additional context**
Output: 
Error: No prior found. Please include at least one.

**Comment by bouthilx** [comment]
Hi @lavoiems! Just to be sure, is this for arguments containing multiple characters but a single dash, like `-this`?

**Comment by lavoiems** [comment]
The argument contains no dashes. Like: `argument: value`

**Comment by bouthilx** [comment]
So something like `python myscript.py argument=value`? Thanks for clarifying, that's not what I had in mind.

**Comment by bouthilx** [comment]
@Delaunay Would the plugin for Hydra solve that issue?

**Comment by Delaunay** [comment]
Yes, if you use hydra, it will be best to use Orion's own plugin https://github.com/facebookresearch/hydra/pull/2050
You can install it with `pip install --upgrade "hydra-orion-sweeper @ git+https://github.com/Delaunay/hydra.git@orion#subdirectory=plugins/hydra_orion_sweeper"`


You can find an example of use [here](https://github.com/facebookresearch/hydra/tree/24c7ecba19a9404c7ec5366dfdbd43c2ebb5bcee/plugins/hydra_orion_sweeper/example)

You will able to specify arguments the hydra way and interface with orion as the sametime.

**Comment by lavoiems** [comment]
I tried using this plugin, but it has a dependency on Hydra that is in conflict with fairseq (https://github.com/pytorch/fairseq), another framework that I use.

Fairseq requires `"hydra-core>=1.0.7,<1.1"` where as Orion's own plugin requires `hydra-core>1.1`.
